atsign-foundation/atsign.dev-3.0
# fix: Redirecting Issue in the Flutter documentation

**- What I did**
I updated the Links in the Flutter Get Started Section in the documentation at - https://docs.atsign.com/start/flutter/ .
They were earlier redirecting on the same page but now they open up on a new page.
**- How I did it**
I changed the links from the traditional MarkDown links to HTML anchor Tags to use the target attribute of the anchor tag to redirect to the new page.

**- How to verify it**
Open the Flutter portion of the Get Started section and click any of the available links to verify that they open on a new page.
![image](https://user-images.githubusercontent.com/76838551/193342853-97e4e035-b616-43fc-9877-702ca7abf50e.png)
The yellow text in the above image are all links. 

**- Description for the changelog**
This PR includes the rectified functionality of the Links present on the flutter portion of the Get Started section. They were earlier redirection on the same page after these changes they will redirect to a new page.
